ui/narrative/methods/export_json_to_staging/spec.json

```diff
@@ -0,0 +1,76 @@
+{
+  "ver" : "",
+  "authors" : ["gaprice"],
+  "contact" : "https://www.kbase.us/support/",
+  "visible" : true,
+  "categories" : ["active","util"],
+  "widgets": {
+    "input": null,
+    "output" : "no-display"
+  },
+  "parameters" : [
+    {
+      "id" : "input_ref",
+      "optional" : false,
+      "advanced" : false,
+      "allow_multiple" : false,
+      "default_values" : [ "" ],
+      "field_type" : "text",
+      "text_options" : {
+        "valid_ws_types" : ["*"]
+      }
+    },
+    {
+      "id" : "destination_dir",
+      "optional" : false,
+      "advanced" : false,
+      "allow_multiple" : false,
+      "default_values" : [ "workspace_export" ],
+      "field_type" : "text"
+    },
+    {
+      "id": "format",
+      "optional" : false,
+      "advanced" : false,
+      "allow_multiple" : false,
+      "default_values" : [ "legacy_data_import_export" ],
```

Comment:
we want default to be `legacy` not `standard`?

Reply:
Legacy due to the principle of least surprise